client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -14,9 +15,25 @@ interface DetailTabsProps {
   tabs: TabDefinition[];
 }
 export const DetailTabs: FC<DetailTabsProps> = ({ tabs }) => {
+  const { urlQuery, updateQuery } = useUrlQuery();
   const [currentTab, setCurrentTab] = useState<string>(tabs[0]?.value ?? '');
   const t = useTranslation('common');
 
+  const onChange:
+    | ((event: React.SyntheticEvent<Element, Event>, value: any) => void)
```

Comment:
can this be just `const onChange = (_: React.SyntheticEvent, tab: string) => {` ?

Reply:
thanks! yes, well spotted.